ZConfig/info.py

```diff
@@ -228,6 +220,8 @@ def __init__(self, name, sectiontype, minOccurs, maxOccurs, handler,
         # handler     - handler name called when value(s) must take effect,
         #               or None
         # attribute   - name of the attribute on the SectionValue object
+        assert maxOccurs is not None
+        # because we compare it to 1 which is a Py3 error
```

Comment:
I think the weirdness is partially because this is a subclass of an abstract class that allows subclasses to define the constructors differently (which they do, passing constants for some of the arguments). That comparison works on Py2, but fails on Py3.

Once we do the assert in the constructor, getdefault doesn't matter anymore?